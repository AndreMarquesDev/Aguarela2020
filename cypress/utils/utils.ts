import { Locale } from '../../types/Locale';
import {
    menuIconButtonDataTestId,
    nukaCarouselNextButtonDataTestId,
} from '../../utils/dataTestIds';
import { TextsInterface, textsPt, textsEn } from '../../utils/texts';

export const getLocalizedTexts = (locale: Locale): TextsInterface => {
    if (locale === Locale.Pt) {
        return textsPt;
    }

    return textsEn;
};

export const openMenuMobile = (isMobile: boolean, isFirefox: boolean): void => {
    if (isMobile) {
        cy.getByDataTestId(menuIconButtonDataTestId).click({ force: isFirefox });
    }
};

export const clickNextArrowButtonIfMobile = (
    isMobile: boolean,
    container: string,
    containerTitle: string
): void => {
    const isFirefox = Cypress.isBrowser('firefox');

    if (isMobile) {
        cy.getByDataTestIdParent(container, nukaCarouselNextButtonDataTestId).click({
            force: isFirefox,
        });

        cy.getByText(container, containerTitle).scrollIntoView();
    }
};
