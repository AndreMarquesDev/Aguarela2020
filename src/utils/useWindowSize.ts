import { useEffect, useState } from 'react';

type WindowSize = {
    width: number;
    height: number;
};

export const useWindowSize = (): WindowSize => {
    const [windowSize, setWindowSize] = useState<WindowSize>({
        width: 0,
        height: 0,
    });

    useEffect(() => {
        const handleResize = (): void => {
            // eslint-disable-next-line react-hooks-extra/no-direct-set-state-in-use-effect
            setWindowSize({
                width: window.innerWidth,
                height: window.innerHeight,
            });
        };

        window.addEventListener('resize', handleResize);

        handleResize();

        return (): void => window.removeEventListener('resize', handleResize);
    }, []);

    return windowSize;
};

export enum Breakpoint {
    Desktop = 1280,
    Phablet = 1023,
    Tablet = 767,
    Mobile = 511,
}
