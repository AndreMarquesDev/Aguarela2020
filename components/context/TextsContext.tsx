import { createContext, Context } from 'react';
import { TextsInterface, textsPt } from '../../utils/texts';

export interface TextsContextProps {
    texts: TextsInterface;
}

export const TextsContext: Context<TextsContextProps> = createContext({
    texts: textsPt,
});
