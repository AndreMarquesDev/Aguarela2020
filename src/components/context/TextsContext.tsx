import type { Context } from 'react';
import type { TextsInterface } from '../../utils/texts';
import { createContext } from 'react';
import { textsPt } from '../../utils/texts';

export type TextsContextProps = {
    texts: TextsInterface;
};

export const TextsContext: Context<TextsContextProps> = createContext({
    texts: textsPt,
});
