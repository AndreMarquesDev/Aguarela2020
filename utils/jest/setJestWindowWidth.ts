import { Breakpoint } from '../useWindowSize';

export const setJestWindowWidth = (breakpoint: Breakpoint): void => {
    let width;

    if (breakpoint === Breakpoint.Mobile) {
        width = Breakpoint.Mobile - 1;
    } else if (breakpoint === Breakpoint.Phablet) {
        width = Breakpoint.Phablet - 1;
    } else {
        width = Breakpoint.Desktop + 1;
    }

    Object.defineProperty(window, 'innerWidth', {
        writable: true,
        configurable: true,
        value: width,
    });
};
