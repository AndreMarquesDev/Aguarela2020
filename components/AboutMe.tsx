import React, { FC, useContext } from 'react';
import Image from 'next/image';
import Button from './Button';
import TextsContext from './context/TextsContext';
import Title from './Title/Title';
import { useWindowSize, desktopBreakpoint } from '../utils/useWindowSize';

const AboutMe: FC = () => {
    const { texts } = useContext(TextsContext);
    const windowSize = useWindowSize();
    const isDesktop = windowSize.width > desktopBreakpoint;

    return (
        <>
            <section className="container">
                <div className="wrapper genericMargins">
                    <Title colored text={texts.about} />
                    <div className="mainContentContainer">
                        <div className="imageBlock">
                            <Image
                                priority
                                alt="homepage banner"
                                height={650}
                                src="/images/about-me.jpg"
                                width={530}
                            />
                        </div>
                        <div className="textBlock">
                            <p className="bodyText">{texts.hiMyNameIs}</p>
                            <p className="bodyText">{texts.withMoreThan}</p>
                            <p className="bodyText">{texts.thesocialMediaCommunicationStrategy}</p>
                            <Button
                                alignLeft={isDesktop}
                                externalLink="https://www.instagram.com/catarinasantiago"
                            >
                                {texts.getToKnowMeBetter}
                            </Button>
                        </div>
                    </div>
                </div>
            </section>

            <style jsx>
                {`
                    @import './styles/_vars.scss';

                    .container {
                        background: $white;

                        .mainContentContainer {
                            display: flex;
                            justify-content: space-between;
                            align-items: center;

                            @include desktop {
                                flex-direction: column;
                                justify-content: center;
                            }
                        }

                        .imageBlock {
                            width: 35%;
                            position: relative;

                            &:before {
                                content: '';
                                width: 100%;
                                height: 100%;
                                position: absolute;
                                top: 12rem;
                                left: 12rem;
                                background: $pink;
                            }

                            @include desktop {
                                width: 45%;
                                margin-bottom: 50rem;
                            }

                            @include tablet {
                                width: 75%;
                            }

                            @include mobile {
                                width: 100%;
                            }
                        }

                        .textBlock {
                            width: 60%;

                            p {
                                text-align: left;
                            }

                            @include desktop {
                                width: 80%;

                                p {
                                    text-align: center;
                                }
                            }

                            @include tablet {
                                width: 100%;
                            }
                        }
                    }
                `}
            </style>
        </>
    );
};

export default AboutMe;
