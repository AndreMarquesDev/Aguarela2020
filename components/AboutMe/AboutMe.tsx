import React, { useContext } from 'react';
import Image from 'next/image';
import { Button } from '../Button/Button';
import { TextsContext } from '../context/TextsContext';
import { Title } from '../Title/Title';
import { useWindowSize, Breakpoint } from '../../utils/useWindowSize';
import { aboutMeSectionDataTestId } from '../../utils/dataTestIds';
import { catarinaSantiagoInstagramUrl } from '../../utils/urls';

export const AboutMe = (): JSX.Element => {
    const { texts } = useContext(TextsContext);
    const windowSize = useWindowSize();
    const isDesktop = windowSize.width > Breakpoint.Desktop;

    return (
        <>
            <section className="container" data-testid={aboutMeSectionDataTestId}>
                <div className="wrapper genericMargins">
                    <Title colored text={texts.about} />
                    <div className="mainContentContainer">
                        <div className="imageBlock">
                            <Image
                                priority
                                alt="a picture of me, Catarina"
                                height={650}
                                src="/images/about-me.jpg"
                                width={530}
                            />
                        </div>
                        <div className="textBlock">
                            <p className="bodyText">{texts.hiMyNameIs}</p>
                            <p className="bodyText">{texts.withMoreThan}</p>
                            <p className="bodyText">{texts.theSocialMediaCommunicationStrategy}</p>
                            <Button
                                alignLeft={isDesktop}
                                externalLink={catarinaSantiagoInstagramUrl}
                            >
                                {texts.getToKnowMeBetter}
                            </Button>
                        </div>
                    </div>
                </div>
            </section>

            <style jsx>
                {`
                    @import './styles/_vars.scss';

                    .container {
                        background: $white;

                        .mainContentContainer {
                            display: flex;
                            justify-content: space-between;
                            align-items: center;

                            @include desktop {
                                flex-direction: column;
                                justify-content: center;
                            }
                        }

                        .imageBlock {
                            width: 35%;
                            position: relative;

                            &:before {
                                content: '';
                                width: 100%;
                                height: 100%;
                                position: absolute;
                                top: 12rem;
                                left: 12rem;
                                background: $pink;
                            }

                            @include desktop {
                                width: 45%;
                                margin-bottom: 50rem;
                            }

                            @include tablet {
                                width: 75%;
                            }

                            @include mobile {
                                width: 100%;
                            }
                        }

                        .textBlock {
                            width: 60%;

                            p {
                                text-align: left;
                            }

                            @include desktop {
                                width: 80%;

                                p {
                                    text-align: center;
                                }
                            }

                            @include tablet {
                                width: 100%;
                            }
                        }
                    }
                `}
            </style>
        </>
    );
};
