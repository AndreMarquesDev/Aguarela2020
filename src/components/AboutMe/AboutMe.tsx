import React, { useContext } from 'react';
import Image from 'next/image';
import { Button } from '../Button/Button';
import { TextsContext } from '../context/TextsContext';
import { Title } from '../Title/Title';
import { useWindowSize, Breakpoint } from '../../utils/useWindowSize';
import { aboutMeSectionDataTestId } from '../../utils/dataTestIds';
import { catarinaSantiagoInstagramUrl } from '../../utils/urls';
import { desktopMediaQuery, mobileMediaQuery, tabletMediaQuery } from '../../styles/mediaQueries';

export const AboutMe = (): JSX.Element => {
    const { texts } = useContext(TextsContext);
    const windowSize = useWindowSize();
    const isDesktop = windowSize.width > Breakpoint.Desktop;

    return (
        <>
            <section className="container" data-testid={aboutMeSectionDataTestId}>
                <div className="wrapper genericMargins">
                    <Title colored text={texts.about} />
                    <div className="mainContentContainer">
                        <div className="imageBlock">
                            <Image
                                priority
                                alt="a picture of me, Catarina"
                                height={650}
                                sizes={`(${desktopMediaQuery}) 45vw, (${tabletMediaQuery}) 75vw, (${mobileMediaQuery}) 100vw, 35vw`}
                                src="/images/about-me.jpg"
                                style={{
                                    width: '100%',
                                    height: 'auto',
                                    position: 'relative',
                                }}
                                width={530}
                            />
                        </div>
                        <div className="textBlock">
                            <p className="bodyText">{texts.hiMyNameIs}</p>
                            <p className="bodyText">{texts.withMoreThan}</p>
                            <p className="bodyText">{texts.theSocialMediaCommunicationStrategy}</p>
                            <Button
                                alignLeft={isDesktop}
                                externalLink={catarinaSantiagoInstagramUrl}
                            >
                                {texts.getToKnowMeBetter}
                            </Button>
                        </div>
                    </div>
                </div>
            </section>

            <style jsx>
                {`
                    .container {
                        background: var(--white);

                        .mainContentContainer {
                            display: flex;
                            justify-content: space-between;
                            align-items: center;

                            @media (${desktopMediaQuery}) {
                                flex-direction: column;
                                justify-content: center;
                            }
                        }

                        .imageBlock {
                            width: 35%;
                            position: relative;

                            &:before {
                                content: '';
                                width: 100%;
                                height: 100%;
                                position: absolute;
                                top: 12rem;
                                left: 12rem;
                                background: var(--pink);
                            }

                            @media (${desktopMediaQuery}) {
                                width: 45%;
                                margin-bottom: 50rem;
                            }

                            @media (${tabletMediaQuery}) {
                                width: 75%;
                            }

                            @media (${mobileMediaQuery}) {
                                width: 100%;
                            }
                        }

                        .textBlock {
                            width: 60%;

                            p {
                                text-align: left;
                            }

                            @media (${desktopMediaQuery}) {
                                width: 80%;

                                p {
                                    text-align: center;
                                }
                            }

                            @media (${tabletMediaQuery}) {
                                width: 100%;
                            }
                        }
                    }
                `}
            </style>
        </>
    );
};
