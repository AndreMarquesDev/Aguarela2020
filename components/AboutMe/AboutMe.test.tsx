import '@testing-library/jest-dom';
import React from 'react';
import { render, screen } from '@testing-library/react';
import AboutMe from './AboutMe';
import TextsContext from '../context/TextsContext';
import { textsEn, textsPt } from '../../utils/texts';
import { setJestWindowWidth } from '../../utils/jest/setJestWindowWidth';
import { Breakpoint } from '../../utils/useWindowSize';

describe('<AboutMe />', () => {
    test('renders properly', () => {
        const { container } = render(<AboutMe />);

        expect(screen.getByText(textsPt.about)).toBeInTheDocument();
        expect(screen.getByText(textsPt.hiMyNameIs)).toBeInTheDocument();
        expect(screen.getByText(textsPt.withMoreThan)).toBeInTheDocument();
        expect(screen.getByText(textsPt.thesocialMediaCommunicationStrategy)).toBeInTheDocument();
        expect(screen.getByText(textsPt.getToKnowMeBetter)).toBeInTheDocument();

        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
        expect(container.getElementsByClassName('alignCenter').length).toBe(0);
        expect(container).toMatchSnapshot();
    });

    test('renders properly in English', () => {
        const { container } = render(
            <TextsContext.Provider
                value={{
                    texts: textsEn,
                }}
            >
                <AboutMe />
            </TextsContext.Provider>
        );

        expect(screen.getByText(textsEn.about)).toBeInTheDocument();
        expect(screen.getByText(textsEn.hiMyNameIs)).toBeInTheDocument();
        expect(screen.getByText(textsEn.withMoreThan)).toBeInTheDocument();
        expect(screen.getByText(textsEn.thesocialMediaCommunicationStrategy)).toBeInTheDocument();
        expect(screen.getByText(textsEn.getToKnowMeBetter)).toBeInTheDocument();

        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
        expect(container.getElementsByClassName('alignCenter').length).toBe(0);
        expect(container).toMatchSnapshot();
    });

    test('renders properly in mobile', () => {
        setJestWindowWidth(Breakpoint.Mobile);

        const { container } = render(<AboutMe />);

        // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
        expect(container.getElementsByClassName('alignCenter').length).toBe(1);
        expect(container).toMatchSnapshot();
    });
});
