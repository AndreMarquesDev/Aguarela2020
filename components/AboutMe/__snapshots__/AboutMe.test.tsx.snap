// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutMe /> renders properly 1`] = `
<div>
  <section
    class="jsx-1128069698 container"
    data-testid="aboutMe_section"
  >
    <div
      class="jsx-1128069698 wrapper genericMargins"
    >
      <h1
        class="jsx-3155756322 colored"
      >
        Sobre
      </h1>
      <div
        class="jsx-1128069698 mainContentContainer"
      >
        <div
          class="jsx-1128069698 imageBlock"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTMwIiBoZWlnaHQ9IjY1MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="a picture of me, Catarina"
              decoding="async"
              src="/_next/image?url=%2Fimages%2Fabout-me.jpg&w=1080&q=75"
              srcset="/_next/image?url=%2Fimages%2Fabout-me.jpg&w=640&q=75 1x, /_next/image?url=%2Fimages%2Fabout-me.jpg&w=1080&q=75 2x"
              style="visibility: inherit; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="jsx-1128069698 textBlock"
        >
          <p
            class="jsx-1128069698 bodyText"
          >
            Olá! O meu nome é Catarina Santiago e sou gestora de redes sociais e fotógrafa baseada em Lisboa.
          </p>
          <p
            class="jsx-1128069698 bodyText"
          >
            Com mais de 4 anos e 18 clientes de experiência, um projeto de freelance transformou-se na Aguarela Digital, uma agência de comunicação com o objetivo de posicionar e fazer brilhar marcas e negócios no digital.
          </p>
          <p
            class="jsx-1128069698 bodyText"
          >
            A estratégia de comunicação em redes sociais é sempre desenvolvida com base nos objetivos do cliente. Embora seja importante ser flexível e compreensiva, o meu papel é pensar fora da caixa, criar conteúdo inovador para redes sociais e otimizar a estratégia com base nos resultados e tendências de mercado. Acredito que uma boa estratégia de comunicação aliada a um planeamento visual de conteúdo é o segredo para o aumento da visibilidade e faturação do negócio.
          </p>
          <div
            class="jsx-781296904 buttonWrapper"
          >
            <div
              class="jsx-781296904 buttonBackground"
            >
              <a
                class="jsx-781296904 button"
                href="https://www.instagram.com/catarinasantiago/"
                rel="noreferrer"
                target="_blank"
              >
                Conhece-me melhor
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`<AboutMe /> renders properly in English 1`] = `
<div>
  <section
    class="jsx-1128069698 container"
    data-testid="aboutMe_section"
  >
    <div
      class="jsx-1128069698 wrapper genericMargins"
    >
      <h1
        class="jsx-3155756322 colored"
      >
        About
      </h1>
      <div
        class="jsx-1128069698 mainContentContainer"
      >
        <div
          class="jsx-1128069698 imageBlock"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTMwIiBoZWlnaHQ9IjY1MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="a picture of me, Catarina"
              decoding="async"
              src="/_next/image?url=%2Fimages%2Fabout-me.jpg&w=1080&q=75"
              srcset="/_next/image?url=%2Fimages%2Fabout-me.jpg&w=640&q=75 1x, /_next/image?url=%2Fimages%2Fabout-me.jpg&w=1080&q=75 2x"
              style="visibility: inherit; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="jsx-1128069698 textBlock"
        >
          <p
            class="jsx-1128069698 bodyText"
          >
            Hi! My name is Catarina Santiago and I'm a social media manager and photographer based in Lisbon.
          </p>
          <p
            class="jsx-1128069698 bodyText"
          >
            With more than 4 years of experience and 18 clients, a freelance project became Aguarela Digital, a communication agency with the goal to position and make brands and businesses shine in the digital world.
          </p>
          <p
            class="jsx-1128069698 bodyText"
          >
            The social media strategy is always developed based on the client's goals. While it is important to be flexible, my role is to think outside the box by creating innovative content and optimizing the strategy based on results and market trends. I believe that a good communication strategy combined with a visual planning of the content is the key to increase the visibility and revenue of the business.
          </p>
          <div
            class="jsx-781296904 buttonWrapper"
          >
            <div
              class="jsx-781296904 buttonBackground"
            >
              <a
                class="jsx-781296904 button"
                href="https://www.instagram.com/catarinasantiago/"
                rel="noreferrer"
                target="_blank"
              >
                Get to know me better
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;

exports[`<AboutMe /> renders properly in mobile 1`] = `
<div>
  <section
    class="jsx-1128069698 container"
    data-testid="aboutMe_section"
  >
    <div
      class="jsx-1128069698 wrapper genericMargins"
    >
      <h1
        class="jsx-3155756322 colored"
      >
        Sobre
      </h1>
      <div
        class="jsx-1128069698 mainContentContainer"
      >
        <div
          class="jsx-1128069698 imageBlock"
        >
          <div
            style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
          >
            <div
              style="box-sizing: border-box; display: block; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTMwIiBoZWlnaHQ9IjY1MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
                style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
              />
            </div>
            <img
              alt="a picture of me, Catarina"
              decoding="async"
              src="/_next/image?url=%2Fimages%2Fabout-me.jpg&w=1080&q=75"
              srcset="/_next/image?url=%2Fimages%2Fabout-me.jpg&w=640&q=75 1x, /_next/image?url=%2Fimages%2Fabout-me.jpg&w=1080&q=75 2x"
              style="visibility: inherit; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
          </div>
        </div>
        <div
          class="jsx-1128069698 textBlock"
        >
          <p
            class="jsx-1128069698 bodyText"
          >
            Olá! O meu nome é Catarina Santiago e sou gestora de redes sociais e fotógrafa baseada em Lisboa.
          </p>
          <p
            class="jsx-1128069698 bodyText"
          >
            Com mais de 4 anos e 18 clientes de experiência, um projeto de freelance transformou-se na Aguarela Digital, uma agência de comunicação com o objetivo de posicionar e fazer brilhar marcas e negócios no digital.
          </p>
          <p
            class="jsx-1128069698 bodyText"
          >
            A estratégia de comunicação em redes sociais é sempre desenvolvida com base nos objetivos do cliente. Embora seja importante ser flexível e compreensiva, o meu papel é pensar fora da caixa, criar conteúdo inovador para redes sociais e otimizar a estratégia com base nos resultados e tendências de mercado. Acredito que uma boa estratégia de comunicação aliada a um planeamento visual de conteúdo é o segredo para o aumento da visibilidade e faturação do negócio.
          </p>
          <div
            class="jsx-781296904 buttonWrapper"
          >
            <div
              class="jsx-781296904 buttonBackground alignCenter"
            >
              <a
                class="jsx-781296904 button"
                href="https://www.instagram.com/catarinasantiago/"
                rel="noreferrer"
                target="_blank"
              >
                Conhece-me melhor
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
